tableauserverclient/models/view_item.py

```diff
@@ -1,10 +1,12 @@
 import xml.etree.ElementTree as ET
 from .exceptions import UnpopulatedPropertyError
+from .tagged_resource_item import TaggedResourceItem
 from .. import NAMESPACE
 
 
-class ViewItem(object):
+class ViewItem(TaggedResourceItem):
     def __init__(self):
+        super(ViewItem, self).__init__()  # Python2 compatible super
```

Comment:
Well I didn't! I had done it the way I was used to and only thought to check python2 compat as an afterthought... will remove though since it sounds apparent to others.